### PR TITLE
Added text file

### PR DESCRIPTION
**Pull request by sumansuruchi**
I added Task SMS-9

### CODE REVIEW

**Review by lalitghongade — CHANGES_REQUESTED** [review]
can you added details like comments

**Review by sargarakshay — APPROVED** [review]
Looks Good To Me

**Comment by sumansuruchi** [comment]
> can you added details like comments
added Comments

**Review by lalitghongade — APPROVED** [review]
good to go

File: Suruchi.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+hi everyone working on
```
Comment (sargarakshay) [inline Suruchi.txt:1]: Use proper naming conventions

Comment (sargarakshay) [inline Suruchi.txt:1]:
```suggestion
hi everyone working on SMS-9
```

**Review by sargarakshay — CHANGES_REQUESTED** [review]

**Pull request merged**